js-functions_basic-functions/js/index.js

```diff
@@ -37,5 +46,13 @@ Then log all variables to the console again.
 */
 
 // --v-- write your code here --v--
+function logBookData () {
+  console.log(title, author, rating, numberSales);
```

Comment:
All good, but here a tip to make it look better in the console:

console.log(`Title: ${title}\nAuthor: ${author}\nRating: ${rating}\nSales: ${numberSales}`);

Reply:
Got it. Thank you!